app/uk/gov/hmrc/hmrcemailrenderer/templates/registeryourcompany/registerYourCompanyRegisterVATEmail.scala.txt

```diff
@@ -0,0 +1,14 @@
+@(params: Map[String, Any])You can now register for VAT online
+
+You must register as soon as your VAT taxable turnover:
+
+  is more than £85,000 for any 12-month period
+  is more than £85,000 for a single 30-day period
+
+You can still register voluntarily if the company's VAT taxable turnover is less than £85,000 a year.
+
+To access the service, use the link you received previously in the 'Register your company' welcome email
+
+From HMRC
```

Comment:
Our guidelines at https://github.com/hmrc/hmrc-email-renderer/blob/master/CONTRIBUTING.md recommend not saying just "From HMRC" but to include more detail.  I see the existing "Register Your Company" emails have "From HMRC and Companies House", which would be better.